src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
remove main class

Reply:
I don't understand why I need to delete the main class. Please explain to me